package.json

```diff
@@ -17,7 +17,7 @@
     "flat",
     "object"
   ],
-  "author": "Matteo Collina <hello@matteocollina.com>",
+  "author": "Matteo Collina <hello@matteocollina.com> and Jairus Tanaka <jairus.v.tanaka@outlook.com>",
```

Comment:
Please revert this, maybe add yourself in the collaborators

Reply:
Sure. Do you want to edit it, or should I submit another pull?